src/frontend/shortlist/package.json

```diff
@@ -15,6 +15,7 @@
     "@popperjs/core": "^2.11.6",
     "axios": "^1.1.3",
     "bootstrap": "^5.2.2",
+    "mitt": "^3.0.0",
```

Comment:
mitt is a smart move. Be careful about the Daniel's way. If no conflict, I am good with this.

Reply:
@adelaidemz Do you want to add a comment in the file?

Reply:
The updates are adaptable to accept future changes by Daniel.